server/config_test.go

```diff
@@ -0,0 +1,89 @@
+package server
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/kubernetes-incubator/cri-o/libkpod"
+)
+
+const fixturePath = "fixtures/crio.conf"
+
+func must(t *testing.T, err error) {
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func assertAllFieldsEquality(t *testing.T, c Config) {
+	testCases := []struct {
+		fieldValue, expected interface{}
+	}{
+		{c.RootConfig.Root, "/var/lib/containers/storage"},
+		{c.RootConfig.RunRoot, "/var/run/containers/storage"},
+		{c.RootConfig.Storage, "overlay2"},
```

Comment:
Please use overlay not overaly2,  In containers/storage they are the same.

Reply:
It would probably be better if these wer all set as constants some where.

Reply:
Dan, this value isn't used, this can be changed to "foobar", this test just makes sure whatever in config is read back in the struct (thus avoid also to write consts for these strings)

Reply:
Sure, I just want consistency and no one to make the mistake.  If people read this and see these objects named differently elsewhere it can cause confusion.

Reply:
I've made the change proposed: "overlay2" -> "overlay"

However, I think that moving everything to constants just for the tests is a little bit too much. Maybe I could open an issue and fix it in other PR to change the references to "overlay" string in the whole codebase and use a constant instead? This test included, of course.